source/_components/sensor.geizhals.markdown

```diff
@@ -23,16 +22,16 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
 sensor:
   - platform: geizhals
     name: qc35
-    product_id: 1453021
+    product_id: 1696985
 ```
 
 Configuration variables:
```

Comment:
As you're updating the config block anyway, now would be an ideal time to update this to use the new format as detailed in the documentation standards ;)

Reply:
What do you mean by that?

Reply:
https://developers.home-assistant.io/docs/en/documentation_standards.html#component-and-platform-pages